mainapp/templates/mainapp/modelform.html

```diff
@@ -95,14 +95,31 @@
 		{{ form.scale }}
 		<span id="scaling-help" class="help-block">Specifies the scaling of the model. '2' here would mean that each unit in the model corresponds to 2 meters.</span>
 	</div>
-	<div class="form-group required">
+	<div class="form-group">
+		<label>Where does the model come from?</label>
+		{% for error in form.model_source.errors %}<div class="alert alert-danger" role="alert">{{ error }}</div>{% endfor %}
+		{% for radio in form.model_source %}
+		<div class="radio">
+			<label>{{ radio.tag }}{{ radio.choice_label }}</label>
+		</div>
+		{% endfor %}
+	</div>
+	<div id="model-source" class="form-group">
+		<label for="{{ form.source.id_for_label }}">{{ form.source.label }}:</label>
+		{% for error in form.source.errors %}<div class="alert alert-danger" role="alert">{{ error }}</div>{% endfor %}
+		<div class="alert alert-danger" role="alert" id="source-error" style="display: none;"></div>
+		{{ form.source }}
+		<span id="source-help" class="help-block">Please specify the exact link to the source model.</span>
```

Comment:
Now that it isn't necessarily a link, change to something like:

If possible, specify the exact link to the source model. If the model doesn't come from another website, describe the origin as a text.